### PR TITLE
Add git push notification chat system messages

### PR DESCRIPTION
**Pull request by patrickxb**

### CODE REVIEW

File: go/chat/helper.go [hunk 1]
```diff
@@ -118,6 +118,37 @@ func (h *Helper) SendMsgByNameNonblock(ctx context.Context, name string, topicNa
 	return helper.SendBody(ctx, body, msgType)
 }
 
+func (h *Helper) FindConversations(ctx context.Context, name string, topicName *string, topicType chat1.TopicType, membersType chat1.ConversationMembersType, vis keybase1.TLFVisibility) ([]chat1.ConversationLocal, error) {
+	kuid, err := CurrentUID(h.G())
```
Comment (mmaxim) [inline go/chat/helper.go]: This can probably just call `FindConversations` directly

File: go/chat/helper.go [hunk 2]
```diff
@@ -118,6 +118,37 @@ func (h *Helper) SendMsgByNameNonblock(ctx context.Context, name string, topicNa
...
+	if err != nil {
+		return nil, err
+	}
+	uid := gregor1.UID(kuid.ToBytes())
+	ncHelper := newNewConversationHelper(h.G(), uid, name, topicName, topicType, membersType, vis, h.ri)
```
Comment (mmaxim) [inline go/chat/helper.go]: This can probably just call `FindConversations` directly.

File: go/chat/helper.go [hunk 3]
```diff
@@ -118,6 +118,37 @@ func (h *Helper) SendMsgByNameNonblock(ctx context.Context, name string, topicNa
...
+	convs, _, err := ncHelper.findConversations(ctx, membersType, *topicName)
+	return convs, err
+}
+
+func (h *Helper) FindConversationsByID(ctx context.Context, convIDs []chat1.ConversationID) ([]chat1.ConversationLocal, error) {
+	kuid, err := CurrentUID(h.G())
+	if err != nil {
+		return nil, err
+	}
+	uid := gregor1.UID(kuid.ToBytes())
```
Comment (mmaxim) [inline go/chat/helper.go]: Same with this.

File: go/git/put.go [hunk 4]
```diff
@@ -55,5 +57,55 @@ func PutMetadata(ctx context.Context, g *libkb.GlobalContext, arg keybase1.PutGi
 		},
 	}
 	_, err = g.GetAPI().Post(apiArg)
+
+	if err == nil {
+		err = sendChat(ctx, g, teamIDVis.TeamID, arg)
```
Comment (mmaxim) [inline go/git/put.go]: Maybe go this on goroutine?

Comment (mmaxim) [reply go/git/put.go]: Or use nonblock version of send message.

Comment (mmaxim) [reply go/git/put.go]: (I don't care too much about this)

Comment (patrickxb) [reply go/git/put.go]: Changed to use nonblock.

**Pull request merged**